src/refdb_backend.c

```diff
@@ -198,7 +198,6 @@ pygit2_refdb_backend_lookup(git_reference **out,
 
     *out = result->reference;
 out:
-    Py_DECREF(result);
```

Comment:
Why this? Doesn't `lookup(..)` return a new reference?

Reply:
Apparently not. This was causing double frees.